src/MoBi.Core/Domain/Model/MoBiSimulation.cs

```diff
@@ -122,6 +133,24 @@ public IReadOnlyList<IBuildingBlock> BuildingBlocks()
          return buildingBlocks;
       }
 
+      public IReadOnlyList<ParameterValue> OriginalQuantityValues => _quantityValueCache.ToList();
+
+      public void AddOriginalQuantityValue(ParameterValue parameterValue)
+      {
+         if (_quantityValueCache[parameterValue.Path] == null)
```

Comment:
why the check here?. Shoudn't you always overwrite?

Reply:
Ok I see. You only want to add it once. This needs to be explained here that we want to track the very first entry.